src/fetching/tasks/getTaskFilterSpec.ts

```diff
@@ -0,0 +1,10 @@
+import { defaultFetch } from '..';
+import { ZetkinSmartSearchFilter } from '../../types/zetkin';
+
+export default function getTaskFilterSpec(orgId : string, taskId: string, fetch = defaultFetch) {
+    return async () : Promise<ZetkinSmartSearchFilter[]> => {
+        const res = await fetch(`/orgs/${orgId}/tasks/${taskId}/target`);
+        const body = await res.json();
+        return body.data.filter_spec;
```

Comment:
not super keen on retrieving just the filter_spec as opposed to the whole target but -- 
1. what else can be in the target?
2. I did it this way to use the `ZetkinSmartSearchFilter` type but I could also just type `ZetkinTarget`

Reply:
The "target" is a Smart Search (in the API called "query") object, i.e. it has an `id` and a `filter_spec`. You can use the ID of a query to work with it in the API at `/orgs/{org_id}/people/queries/{query_id}`.

This is likely how we will want to interact with a query from this suite of UIs eventually, because we want to use the same UI for all types of Smart Search queries. But it's not important for MVP.